api/python/tests/test_data_transfer.py

```diff
@@ -431,9 +433,15 @@ def test_multipart_copy(self):
         )
 
         with mock.patch('quilt3.data_transfer.s3_transfer_config.max_request_concurrency', 1):
-            data_transfer.copy_file_list([
-                (PhysicalKey.from_url('s3://example1/large_file1.npy'), PhysicalKey.from_url('s3://example2/large_file2.npy'), size),
-            ])
+            stdout = io.StringIO()
+
+            # temporarily redirect stderr to capture warnings (usually errors)
```

Comment:
Not quite sure how this comment is relevant, probably it should explain that by default `tqdm` writes to `stderr`.